css/index.css

```diff
@@ -0,0 +1,376 @@
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+body {
+  line-height: 1;
+}
+ol,
+ul {
+  list-style: none;
+}
+blockquote,
+q {
+  quotes: none;
+}
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: '';
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+* {
+  box-sizing: border-box;
+}
+html {
+  font-size: 62.5%;
+}
+html,
+body {
+  font-family: 'Ubuntu', sans-serif;
+  color: #5E6164;
+}
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: 'Russo One', sans-serif;
+  color: #23293B;
+}
+h2 {
+  font-size: 3.2rem;
+}
+h3 {
+  font-size: 2.8rem;
+}
+p {
+  font-size: 1.6rem;
+  line-height: 1.4;
+}
+.container {
+  max-width: 800px;
+  width: 100%;
+  margin: 0 auto;
+}
+.nav-bar {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  background-color: #F6F6F6;
+  border-bottom: #F1F1F1;
+}
+.nav-bar h1 {
+  font-size: 4rem;
+  color: #23293B;
+}
+.nav-bar nav {
+  display: flex;
+  justify-content: center;
+  margin: 15px 2rem;
+}
+.nav-bar nav a {
+  color: black;
+  font-family: #454A59;
+  text-decoration: none;
+  font-size: 2rem;
+  margin: 10px 2rem;
+}
+@media (max-width:500px) {
+  .nav-bar nav a {
+    font-size: 1.5rem;
+  }
+}
+footer {
+  color: #22283B;
+  background: #f4f4f4;
+  padding: 20px 0;
+  border: 1px double #e0e0e0;
+  border-radius: 10px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.intro h2 {
+  padding: 15px;
+}
+.intro p {
+  padding: 0 15px;
+}
+.intro .header-image {
+  border-radius: 4px;
+}
+.space-images {
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+}
+@media (max-width:500px) {
+  .space-images .blast {
+    width: 92%;
+    margin: 5% 5%;
+  }
+}
+@media (max-width:500px) {
+  .space-images .sky,
+  .space-images .launch {
+    display: none;
+  }
+}
+.space-images img {
+  width: 28%;
+  margin: 20px 2.5%;
+  border-radius: 20px;
+  height: 100%;
+}
+@media (max-width:500px) {
+  .space-images img {
+    width: 92%;
+    margin: 0 5%;
+  }
+}
+.visit-planets {
+  padding: 15px;
+}
+.visit-planets h3 {
+  margin-top: 11px;
+}
+.visit-planets p {
+  margin-top: 13px;
+}
+.walk {
+  margin-top: 60px;
+  border-top: 1px solid #dee2e6;
+  padding: 50px 60px;
+  display: flex;
+  justify-content: center;
+}
+@media (max-width:500px) {
+  .walk {
+    flex-direction: column-reverse;
+    margin-top: 10px;
+    padding: 10px 5%;
+  }
+}
+.walk .walk-text {
+  width: 100%;
+}
+@media (max-width:500px) {
+  .walk .walk-text {
+    width: 100%;
+    margin-top: 10px;
+  }
+}
+.walk .walk-text h3 {
+  margin-bottom: 10px;
+}
+.walk .walk-text p {
+  margin-bottom: 26px;
+}
+.walk .walk-text .btn {
+  background-color: #22283A;
+  color: #F6F7F8;
+  width: 30%;
```

Comment:
setting the width to 30%, your text on the button jumps to a new line.  setting it to a hard pixel like 100px takes away that problem.